logconv.m

```diff
@@ -23,7 +23,7 @@ function logconv()
 % ************************************************************************
 
 % set the path to your log file here file here
-fileName = 'log001.ulg';
```

Comment:
should be changed back to the default value

Reply:
Already changed back